packages/profiles/source/transaction.aggregate.test.ts

```diff
@@ -1,4 +1,5 @@
 import { describe } from "@payvo/sdk-test";
+import { Contracts } from ".";
```

Comment:
Import what you need from a specific file.

Reply:
i blame vs code for that :colbert: